### PR TITLE
g-music as input provider

### PR DESCRIPTION
**Pull request by alexshavelev44**
issue - https://github.com/Ignisor/music-share-bot/issues/16

### CODE REVIEW

File: core/providers/google_music.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+
+from core.providers.base import MusicProvider
+
+class GoogleMusic(MusicProvider):
+    NAME = 'Google Music'
+
+    @classmethod
+    def is_music_url(self, url):
+        if 'https://play.google.com/music' in url:
```
Comment (Ignisor) [inline core/providers/google_music.py]: Are you Shure that `https` will be present in all links?

File: core/providers/google_music.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    def get_music_url(self, name):
+        pass
+
+
+    def get_music_name(self, url):
+        g_music_page = requests.get(url)
+        soup = BeautifulSoup(g_music_page.content, 'html.parser')
+        title = soup.find('div', class_='title fade-out').text
+        artist = soup.find('div', class_='album-artist fade-out').text
+        return f'{artist} - {title}'
```
Comment (Ignisor) [inline core/providers/google_music.py]: No newline at end of file

**Comment by alexshavelev44** [comment]
@Ignisor fixed

**Comment by Ignisor** [comment]
@alexshavelev44 You forgot to add beatifulsoup to requirements and update readme. And I forgot to check ;)

I handled it

**Comment by alexshavelev44** [comment]
after install soap and freeze requirements file looks strange 
image

**Comment by Ignisor** [comment]
@alexshavelev44 You can use `pipreqs` or add that line manually

**Pull request merged**